README.md

```diff
@@ -46,6 +46,15 @@ Active namespace is "kube-system".
 $ kubens -
 Context "test" set.
 Active namespace is "default".
+
+# change the active namespace even if it doesn't exist
+$ kubens not-found-namespace --force
+Context "test" set.
+Active namespace is "not-found-namespace".
+---
+$ kubens not-found-namespace --f
```

Comment:
Double dash is weird to see for single letter flags. Mistake?

Reply:
yes, it's mistake, pushed fix